config.json

```diff
@@ -2,5 +2,6 @@
   "SWAP_DIRECTORY": "/tmp/",
   "LOGFILE": "/tmp/output.log",
   "PORT": "8000",
+  "DBDRIVER": "memory",
```

Comment:
I don't think so, because I want in the future to add support to Redis or another database.